### PR TITLE
Update the Visio Importer documentation

### PR DESCRIPTION
**Pull request by FrankeLukas**
Adds documentation for registration and usage.
Removes old entries and adds references.

### CODE REVIEW

File: docs/admin/services/visio-importer/login.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+
+Before you can start importing Visio files to Symbio, please make sure have your PIN at hand.
+
+![login](media/login.png)
+
+Enter your PIN and click on "**Confirm**". If you do not have a PIN yet, please start the registration process on the [Homepage](https://visio.symbioweb.com/) by clicking on "**Register here**". The registration itself requires a valid email address and your agreement to the general [Terms and Services](https://visio.symbioweb.com/nutzungsbedingungen.html).
+
+![register](media/register.png)
+
+You will receive an email containing the PIN after you registered successfully. Please contact support if you didn’t receive an email.
```
Comment (JochenLinnemann) [inline docs/admin/services/visio-importer/login.md:11]: Shouldn't we add a support link / e-mail address here?

**Review by JochenLinnemann — CHANGES_REQUESTED** [review]
Can you add screenshots of the Administrative Area without "(Debug)" in the title?

File: docs/admin/services/visio-importer/login.md (hunk 1 above)
Comment (FrankeLukas) [reply docs/admin/services/visio-importer/login.md]: Added a link to https://www.symbioworld.com/en/service/support/.

**Comment by FrankeLukas** [comment]
Removed `(Debug)` from the images and added a reference to https://www.symbioworld.com/en/service/support/.

**Review by JochenLinnemann — APPROVED** [review]

**Comment by FrankeLukas** [comment]
@JochenLinnemann I fixed the review comments from Ioana so this is good to go now.

**Review by JochenLinnemann — APPROVED** [review]

**Pull request merged**